internal/admin/static/app.css

```
@@ -355,185 +485,418 @@ header {
+  gap: 8px;
+}
+/* Build identity, in mono because operators quote it verbatim into bug
+ * reports and a proportional font makes a hash hard to transcribe. */
+.build-line {
+  margin: 0;
+  font-family: var(--mono);
+  font-size: var(--text-micro);
+  letter-spacing: 0.01em;
+  color: var(--fg-muted);
+  opacity: 0.8;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)